mne/surface.py

```diff
@@ -751,6 +759,38 @@ def _create_surf_spacing(surf, hemi, subject, stype, ico_surf, subjects_dir):
     return surf
 
 
+def _decimate_surface_spacing(s, spacing):
```

Comment:
why don't you call this `surf` as elsewhere? didn't we discourage single letter variable names? Because they are hard to Ctrl + F and also create issues in pdb.